icepyx/core/APIformatting.py

```diff
@@ -352,7 +353,7 @@ def build_params(self, **kwargs):
             self._check_valid_keys()
 
         if self.partype == "required":
-            if self.check_req_values == True and kwargs == {}:
+            if self.check_req_values() and kwargs == {}:
```

Comment:
Since `check_req_values` is a method this would never evaluate to True. I think this was the intention?

Reply:
Thanks for catching and fixing!